packages/shared/src/components/Feed.tsx

```diff
@@ -225,6 +226,12 @@ export default function Feed<T>({
     'go to link',
   );
 
+  const {
+    onFeedArticleClick: onReadArticleClick,
+    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+  } = useCompanionTrigger(triggerReadArticleClick);
+
```

Comment:
We need to add this to the potential Feed refactor to avoid calling hooks conditionally.  cc @rebelchris